addon-test-support/@ember/test-helpers/dom/double-click.ts

```diff
@@ -80,6 +81,8 @@ export function __doubleClick__(element: Element | Document, options: MouseEvent
   doubleClick('button', { shiftKey: true });
 */
 export default function doubleClick(target: Target, options: MouseEventInit = {}): Promise<void> {
+  const mergedOptions = assign({ buttons: 1, button: 0 }, options);
```

Comment:
Same as my previous comment above re: variables to represent the numbers.

Reply:
Added a `-constants` file